requirements/extras/huggingface_requirements.txt

```diff
@@ -0,0 +1,7 @@
+accelerate
```

Comment:
Quite a few of this is already included as part of setup.py, I'd suggest compare against that and revise this section.

Reply:
Ack wil do, thanks